### PR TITLE
DOCS-1718: Publishes OSS 3.26.3

### PR DESCRIPTION
**Pull request by ctauchen**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico_versioned_docs/version-3.26/releases.json [hunk 1]
```diff
@@ -1,4 +1,56 @@
 [
+  {
+    "title": "v3.26.2",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "registry": "quay.io",
+      "version": "v1.30.3"
```
Comment (ctauchen) [inline calico_versioned_docs/version-3.26/releases.json]: @danudey Have you got a new operator version for us? 

And any other component version changes while you're at it? Thanks!

File: calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.2-release-notes.mdx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+29 September 2023
+
+#### Bug fixes
```
Comment (ctauchen) [inline calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.2-release-notes.mdx]: @danudey Have you got one of those nifty autogenerated RNs for us?

**Comment by ctauchen** [comment]
FYI @mgleung

File: calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.2-release-notes.mdx (hunk 2 above)
Comment (ctauchen) [reply calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.2-release-notes.mdx]: Updated with notes from @mgleung

File: calico_versioned_docs/version-3.26/releases.json (hunk 1 above)
Comment (ctauchen) [reply calico_versioned_docs/version-3.26/releases.json]: Updated

File: calico_versioned_docs/version-3.26/variables.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ const variables = {
   noderunning: 'calico-node',
   rootDirWindows: 'C:\\CalicoWindows',
   ppa_repo_name: 'calico-3.26',
-  manifestsUrl: 'https://raw.githubusercontent.com/projectcalico/calico/v3.26.1',
+  manifestsUrl: 'https://raw.githubusercontent.com/projectcalico/calico/v3.26.2',
```
Comment (mgleung) [inline calico_versioned_docs/version-3.26/variables.js]: This should probably be 3.26.3 too

**Pull request merged**